tests/pipelines/main.nf.test

```diff
@@ -38,10 +47,23 @@ nextflow_pipeline {
             assert actual_tree.text == expected_tree.text
             assert actual_clusters.text ==  expected_clusters.text
 
+            // Check appended metadata is correct:
+            def actual_metadata = path("$launchDir/results/append/clusters_and_metadata.tsv")
+            assert actual_metadata.exists()
+            def expected_metadata = path("$baseDir/tests/data/append/expected_clusters_and_metadata.tsv")
+            assert actual_metadata.text == expected_metadata.text
 
             // Check that the ArborView output is created
             def actual_arbor_view = path("$launchDir/results/ArborView/clustered_data_arborview.html")
             assert actual_arbor_view.exists()
+            assert actual_arbor_view.text.contains("myheader_1")
```

Comment:
I don't think these asserts check to make sure that the appropriate data is aligned with the appropriate header. Could you modify these to check for that situation?

Reply:
Updated in 1c6fd666cd90f8d7768e524a1d1a41f261bfb739